components/validator/flags/flags.go

```diff
@@ -126,11 +128,11 @@ var requiredFlags = []cli.Flag{
 
 var optionalFlags = []cli.Flag{
 	AllowNonFinalizedFlag,
-	OutputSubmitterDisabledFlag,
+	OutputSubmitterEnabledFlag,
```

Comment:
Please move these required flags to `requiredFlags`, and move `ProverGrpcFlag` to here.